chart/openfaas/templates/psp.yaml

```diff
@@ -13,27 +13,35 @@ metadata:
         seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
 spec:
     privileged: false
-    hostIPC: false
+    allowPrivilegeEscalation: false
+    requiredDropCapabilities:
+    - ALL
+    volumes:
+    - configMap
+    - downwardAPI
+    - emptyDir
+    - persistentVolumeClaim
+    - projected
+    - secret
     hostNetwork: false
+    hostIPC: false
     hostPID: false
-    readOnlyRootFilesystem: false
-    allowPrivilegeEscalation: false
-    allowedCapabilities:
-        - NET_ADMIN
-        - NET_RAW
```

Comment:
This breaks Istio and Linkerd support since both of them need NET_ADMIN and NET_RAW for the proxy sidecar.

Reply:
Thank you for looking at this. Glad that you had time to review it.

Reply:
As far as I understand, there is nothing preventing people from either setting `psp` to `false` so that this doesn't get created or creating an additional pod security policy that supports Istio/Linkerd.

Reply:
@stefanprodan @alexellis do you have any further concerns regarding this?